local-cli/runWpf/runWpf.js

```diff
@@ -1,45 +1,33 @@
 'use strict';
 
 const chalk = require('chalk');
-const build = require('./utils/build');
-const deploy = require('./utils/deploy');
+const automatedVS = require('../automated-vs/index');
+const WPFSolution = automatedVS.WPFSolution;
+const buildAndDeployUtils = require('../utils/buildAndDeployUtils');
+
+const minWinSDK = '8.1.0.0';
 
```

Comment:
Not sure if this is also supposed to work with Windows 7...

Reply:
I'm not sure either, but this was the original version requirement for wpf. @sayar?